### PR TITLE
2021q2: lldb

### PR DESCRIPTION
**Pull request by mgorny**
CC @emaste

### CODE REVIEW

File: 2021q2/lldb.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
+=== LLDB Debugger Improvements
+
+Links: +
```
Comment (emaste) [inline 2021q2/lldb.adoc:3]: Seems to be some formatting issues with the links in the rendered version (no explicit newlines between each)

I just looked at GitHub's rendering at https://github.com/mgorny/freebsd-quarterly/blob/2021q2-lldb/2021q2/lldb.adoc

**Review by emaste — COMMENTED** [review]
Other than the little nit with the URL list rendering looks good. I'll try to find canonical markup for links for the quarterly reports.

File: 2021q2/lldb.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+link:https://www.moritz.systems/blog/lldb-freebsd-cpu-target-support-and-userland-debugging-improvements/[Moritz Systems Project Description] URL: link:https://www.moritz.systems/blog/lldb-freebsd-cpu-target-support-and-userland-debugging-improvements/[https://www.moritz.systems/blog/lldb-freebsd-cpu-target-support-and-userland-debugging-improvements/]+
```
Comment (jrtc27) [inline 2021q2/lldb.adoc:4]:
```suggestion
link:https://www.moritz.systems/blog/lldb-freebsd-cpu-target-support-and-userland-debugging-improvements/[Moritz Systems Project Description] URL: link:https://www.moritz.systems/blog/lldb-freebsd-cpu-target-support-and-userland-debugging-improvements/[https://www.moritz.systems/blog/lldb-freebsd-cpu-target-support-and-userland-debugging-improvements/] +
```
should fix the rendering; the rest are ok

Comment (mgorny) [reply 2021q2/lldb.adoc]: Merged, thanks!

**Pull request closed without merging**